keepalived/include/snmp.h

```diff
@@ -46,9 +46,10 @@ int header_generic(struct variable *, oid *, size_t *, int,
 
 #define SNMP_DEFAULT_NETWORK_SOCKET	"udp:localhost:705"
 
-#define KEEPALIVED_OID 1, 3, 6, 1, 4, 1, 9586, 100, 5
-#define SNMPTRAP_OID 1, 3, 6, 1, 6, 3, 1, 1, 4, 1, 0
+#define KEEPALIVED_OID {1, 3, 6, 1, 4, 1, 9586, 100, 5}
```

Comment:
In principle I'm not happy with this, because it result in a definition of `GLOBAL_OID` as `{{1, 3, 6, 1, 4, 1, 9586, 100, 5}, 1}` which doesn't seem right.

Could you provide details of the compiler error you are getting, and the version of the compiler, so I can see if I can reproduce it.

I haven't included this change in my updated patch.